### DISCUSSION TITLE
The URBANopt Simulation failed to run ERROR!

### DISCUSSION THREAD

**Discussion by AmirJebraili** [urbanopt/urbanopt-cli#437] [Q&A]
Hey there, when I try to run "DF Run URBANopt" component, It give me the following error.
![01](https://github.com/urbanopt/urbanopt-cli/assets/43678563/848a9f6c-fd52-4aaa-90c1-7682da545b89)

"1. Solution exception:The URBANopt simulation failed to run.
No results were found at:
C:\Users\DearUser\simulation\Urban\run\honeybee_scenario"

These are my softwares and plugins details:
Rhino Version: 7
Ladybug Version: 1.6
URBANopt Version: 0.9.0

I also tried to update my ladybug but when using Versioner component, it doesn't update and shows the following message:
![02](https://github.com/urbanopt/urbanopt-cli/assets/43678563/ccffbd55-e82b-4c56-b2b8-34611353d2b8)

I would appreciate it if anyone can help me solve this issue with URBANopt.

**Comment by kflemin** [comment]
For Ladybug tools support, we recommend posting in the Ladybug Tools forum: https://discourse.ladybug.tools/
